package.json

```diff
@@ -90,25 +90,19 @@
       "/.module-cache/",
       "/react/build/"
     ],
-    "persistModuleRegistryBetweenSpecs": true,
     "rootDir": "",
     "scriptPreprocessor": "scripts/jest/preprocessor.js",
-    "setupEnvScriptFile": "scripts/jest/environment.js",
+    "setupFiles": [ "scripts/jest/environment.js" ],
```

Comment:
Just did `npm install --save-dev some-random-module` and reprinted it correctly. I had no idea this was a thing, TIL :)